commands/commands.json

```
@@ -2,7 +2,7 @@
     "commands": [{
         "name": "new",
         "descripttion": "Create a new DevExtreme application",
-        "usage": "devexterme new <application-type> <name> [options]",
+        "usage": "devextreme new <application-type> <name> [options]",
         "arguments": [{
             "name": "name",
             "descripttion": "The application name"
```

If you started dealing with typos, let's fix another one :)
```suggestion
            "description": "The application name"
```

---

done